libweb3jsonrpc/AdminEth.cpp

```diff
@@ -13,7 +13,7 @@ using namespace dev;
 using namespace dev::rpc;
 using namespace dev::eth;
 
-bool isHex(std::string const& _s)
+bool static isHex(std::string const& _s)
```

Comment:
since it's a .cpp file you can get rid of `std::` prefix by `using namespace std;`